config/README.md

```diff
@@ -0,0 +1,8 @@
+# Config
+
+This directory contains all configuration for both the client and server of the Strimzi-ui ([described here](../docs/Architecture.md#configuration-and-feature-flagging)).
+
+- `runtime.js` - will contain all configuration options which require resolution at server runtime to return a value
```

Comment:
Should these be typescript files now?

Reply:
They should! I am working https://github.com/matthew-chirgwin/strimzi-ui/tree/16-ui-server-exp on what the total convertion of the repo to TS would look like